### PR TITLE
Upgrade to Python3.11

### PR DESCRIPTION
**Pull request by jonathanmetzman**
1. Change Docker images to use Python3
2. Change local scripts to support Python3
3. Fix installation of Chromium dependencies
4. Remove profiling code, the profiling package only supports up to Python3.9
5. Recompile all protos and change some uworker serialization code that has to work differently due to some changes in dependencies
6. Add many no-member pylint exemptions since pylint is now misunderstanding proto members.
7. Change platforms for binary downloads during deploy
8. Upgrade packages that no longer work or have to be upgraded by because of depdency/dependent upgrades
[Used everywhere]
a. grpcio
b. grpcio-tools
c. protobuf
[development]
d. parameterized
e. WebTest
[bots]
f. google-cloud-batch
g. google-cloud-core
h. google-cloud-datastore
i. google-cloud-logging
j. google-cloud-monitoring
k. google-cloud-ndb
l. google-cloud-storage
m. google-api-python-client
n. google-auth
o. google-api-core
[appengine]
p. cryptography
q. firebase-admin
r. PyJWT

9. Remove no longer used pip packages:
a. twisted
b. lxml

### CODE REVIEW

**Comment by jonathanmetzman** [comment]
This can't land until we update Python on the mac bots.

File: cloudbuild.yaml [hunk 1]
```diff
@@ -27,19 +27,19 @@ steps:
     git -C tmp checkout -qf FETCH_HEAD
     mv tmp/.git .git
     rm -rf tmp
-- name: 'gcr.io/clusterfuzz-images/ci'
-  args: ['pipenv', 'sync', '--dev', '--python=3.7']
+- name: 'gcr.io/clusterfuzz-images/ci:5c8f37633ec2f2a1fcbadcb5c6d2711da5a671ad-202307162032'
```
Comment (jonathanmetzman) [inline cloudbuild.yaml:30]: temp change for testing in CI.

**Comment by jonathanmetzman** [comment]
Fixes https://github.com/google/clusterfuzz/issues/3218

**Review by oliverchang — APPROVED** [review]
Awesome stuff! Can we also change our app.yaml (https://github.com/google/clusterfuzz/blob/82bdfee43b675fe7b5b4c92ea23f941353fb9797/configs/test/gae/prod3/app.yaml#L4)

**Review by Alan32Liu — APPROVED** [review]

**Pull request merged**